P9Operators/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿
+// Declaring and initializing the secondsTotal variable.
+int secondsTotal = 0;
+
+// Prompting the user for a number of seconds.
+Console.WriteLine("Give me a number of seconds");
+// Taking the inputut and converting it to int32 then assigning it to the variable secondsTotal.
+secondsTotal = Convert.ToInt32(Console.ReadLine());
+
+// Takes seconds total and divides it by mintues in a day. Then assigns the amount of days to the variable days.
+int days = (int)secondsTotal / 86400;
```

Comment:
Instead of these magic numbers, consider using variables:

```csharp
int minuteToSeconds = 60;
int hourToSeconds = 60*minuteToSeconds;
int dayToSeconds = 24*hourToSeconds;

doubleTotalDaysFraction = (double)totalSeconds / dayToSeconds;
```